### PR TITLE
Migrated payment address.

### PR DESCRIPTION
**Pull request by Elland**
Payment address now comes from the new derivation path, same as Metamask's.

### CODE REVIEW

File: Token/Models/Cereal.swift [hunk 1]
```diff
@@ -81,8 +81,8 @@ public class Cereal: NSObject {
         let idPrivateKey = idKeychain.key.privateKey.hexadecimalString()
         self.idCereal = EtherealCereal(privateKey: idPrivateKey)
 
-        // wallet path: 0H/0/0
-        let walletKeychain = self.mnemonic.keychain.derivedKeychain(at: 0, hardened: true).derivedKeychain(at: 0).derivedKeychain(at: 0)
+        // wallet path: 44H/60H/0H/0
+        let walletKeychain = self.mnemonic.keychain.derivedKeychain(at: 44, hardened: true).derivedKeychain(at: 60, hardened: true).derivedKeychain(at: 0, hardened: true).derivedKeychain(at: 0).derivedKeychain(at: 0)
```
Comment (yuliaveres) [inline Token/Models/Cereal.swift]: why .derivedKeychain(at: 0) twice ?

Comment (yuliaveres) [inline Token/Models/Cereal.swift]: comment says 44H/60H/0H/0

Comment (Elland) [inline Token/Models/Cereal.swift]: Because we need the first key in that path. Should probably update comment.

**Comment by Elland** [comment]
Updated.

**Pull request merged**